File: packages/nativescript-image-cache-it/src-native/imagecacheit/src/main/java/com/github/triniwiz/imagecacheit/EventsListener.java
package com.github.triniwiz.imagecacheit;

import android.graphics.drawable.Drawable;

import androidx.annotation.Nullable;

/**
 * Created by triniwiz on 3/6/20
 */
public interface EventsListener {
    void onLoadStart();
    void onLoadError(String message);
    void onLoadedEnd(@Nullable Drawable drawable);
}

File: packages/nativescript-image-cache-it/src-native/imagecacheit/src/main/java/com/github/triniwiz/imagecacheit/EventsListener.kt
package com.github.triniwiz.imagecacheit

import android.graphics.drawable.Drawable

/**
 * Created by triniwiz on 3/6/20
 */
interface EventsListener {
  fun onLoadStart()
  fun onLoadError(message: String?)
  fun onLoadedEnd(drawable: Drawable?)
}
